### DISCUSSION TITLE
FF7 Remake

### DISCUSSION THREAD

**Discussion by nicomegadrive** [Heroic-Games-Launcher/HeroicGamesLauncher#787] [Q&A]
Hi,

I'm trying to launch the game with different version of Wine / Proton, and enabling DXVK / esynch / fsynch, but when I press the play button or Wine will show me an error and shutdown, or nothing will happen with Proton.

Did someone managed to launch the game and how ?

Here's the log file :
[b25ba0f04dc9434f8a04a062719e7aa7-lastPlay.log](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/files/7729393/b25ba0f04dc9434f8a04a062719e7aa7-lastPlay.log)

Thanks in advance for your help.

**Comment by flavioislima** [comment]
Might be that the game simply doesnt run. 
Do you know if someone was able to make it run?

**Reply by nicomegadrive** [reply]
I managed to make it run with Proton 6.21-GE-2, but it says that I need to launch the game from the real Epic software then close. If there was a way to bypass the check it would be great ;)

Edit : Ok managed to bybass the check by changing the executable but got a black screen (with the music behind) then another error from Direct X 12, will try to get a screenshot.

**Reply by nicomegadrive** [reply]
![erreur-ff7remake](https://user-images.githubusercontent.com/3075241/146427701-3aea3688-2827-4923-af81-ecace590f00d.png)
Here's the error I got.

**Reply by nicomegadrive** [reply]
OK I disabled Feral Game Mode and DXVK install and it works now, will get back if I encounter other problems.

**Reply by flavioislima** [reply]
Ah, nice. Perhaps the way Heroic uses GameMode somehow is launching the game without the epic credentials. 
Will need to test that.

Good to hear that it works!

**Reply by nicomegadrive** [reply]
GameMode is actually fine, I had to disable esynch / fsynch/ dxvk for the game to work.

I'm not sure GameMode is used though on my end, it's already preinstalled on Ubuntu and I just need to write the command lines for it to work on Steam, do you think I need to do the same on your launcher ? Or just enabling GameMode is fine ? (I didn't notice too much change, performance wise)

I played around three hours, and everything work fine from gameplay, to videos, except when there is too much animations (like fire) where it stutters. Must have something to do with my graphic card which isn't optimum but not bad either lol (GTX 1650 SUPER)

Thanks again for your app, it's really cool that I can play this game with it :)

**Reply by flavioislima** [reply]
Yes, Gamemode doesn't improve all games. Actually, I think nowadays, with all improvements Proton, Wine and the Linux Kernel had concerning gaming, it might not be necessary anymore for most of the games. I think just a few have some improvements by maybe 1 or 2%.
But enabling GameMode on the game settings is what you need and I recommend it even if it will give 1 or 2% because its free performance anyway :)

**Comment by DeadlandStrider** [comment]
I got FF7 Remake working by going to the "Other" option in heroics settings and adding "-dx11" without the quotes to "GAME ARGUMENTS" after clicking on the FFVII gear.  
Also, if you are using an AMD GPU you have to use the MESA drivers or you are going to get really bad green light artifacts. 
AMD is updating the mesa driver directly now... so ARCH/Manjaro's AMDGPU driver in the AUR is out of date. If you need openCL for things like blender there is a OpenCL only package you can install. But stick with the Mesa Graphics driver.
